src/platform/packages/shared/kbn-unified-field-list/src/components/field_list_grouped/fields_accordion.tsx

```diff
@@ -69,7 +69,9 @@ function InnerFieldsAccordion<T extends FieldListItem = DataViewField>({
 
     return (
       <EuiText size="xs">
-        <strong className={titleClassname}>{label}</strong>
+        <strong className={titleClassname} aria-label={`${label}: ${fieldsCount} items.`}>
```

Comment:
Can we make it a bit more professional, let's use `i18n` with https://cldr.unicode.org/index/cldr-spec/plural-rules 

Also please don't use '.' for `aria-labels`

example


```
i18n.translate('xxxx', {
        defaultMessage: 'blah blah blah {value, plural, one { one value } other { multiple values }} blah blah',
        values: { value },
      }),
```

Reply:
Improved in [38b8a4e](https://github.com/elastic/kibana/pull/216993#:~:text=i18n%20for%20label-,38b8a4e,-This%20branch%20has)